x-pack/plugins/lists/server/services/exception_lists/get_exception_list_summary.ts

```diff
@@ -84,7 +87,12 @@ export const getExceptionListSummary = async ({
     (acc, item: ByOsAggBucketType) => ({
       ...acc,
       [item.key]: item.doc_count,
-      total: acc.total + item.doc_count,
+      total:
+        // Do not add up the items by OS if host isolation exception
+        // As each host exception entry applies to all OSs
+        listId === 'endpoint_host_isolation_exceptions'
```

Comment:
This is a bit hardcoded... I'm not opposed to not do it but wondering if this is the right place. Could we use an extension point for it? @paul-tavares 
Maybe we could do this list specific transformation in the frontend side

Reply:
@dasansol92 I agree. I couldn't find a substitute when I made the change yesterday so went with this.

I'm in favor of avoiding FE side changes to API data as much as possible; I prefer the data being in a ready-to-use state/shape when arriving on the FE.

Reply:
updated to use a names constant 4f5be8bd8cd8a66a3065288d9b48fe58c22967e3